src/main/java/build/buildfarm/common/config/Worker.java

```diff
@@ -46,14 +46,15 @@ public ExecutionPolicy[] getExecutionPolicies() {
     }
   }
 
-  public WorkerType getWorkerType() {
-    if (getCapabilities().isCas() && getCapabilities().isExecution()) {
-      return WorkerType.EXECUTE_AND_STORAGE;
-    }
+  public int getWorkerType() {
```

Comment:
It would have been nice to preserve the enum so all the caller code was easier to read.  However, I see the value of bitwise flags here as well.

Reply:
If we stick with the bitfield, I'll put better convenience methods (loaders from the packed bits, final properties for the booleans, and packagers to recreate the packed value).